### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by DenishKakadiya**
Added Gujarati translation

### CODE REVIEW

File: locales/gu_IN/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -70,10 +70,11 @@ msgid "%s%s%s Billion Searches"
 msgstr ""
 
 msgctxt "noresults"
-msgid ""
+msgid "કોઈ પરિણામ નહી"
```
Comment (andrey-p) [inline locales/gu_IN/LC_MESSAGES/duckduckgo.po]: The `msgid` is defines the string that needs to be translated - the `""` looks empty but actually it's part of the `"%sClick here%s [...]"` string below. Can you remove the change on this line?

File: locales/gu_IN/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -123,14 +124,14 @@ msgstr ""
 #. at least in:
 #. * https://duckduckgo.com/about. Appears in the window title when you open the link
 msgid "About DuckDuckGo"
-msgstr ""
+msgstr "ડકડકગો વિષે"
```
Comment (andrey-p) [inline locales/gu_IN/LC_MESSAGES/duckduckgo.po]: We prefer to have "DuckDuckGo" untranslated. For example, see: https://github.com/duckduckgo/duckduckgo-locales/blob/2dd90431014eb5e77952d401283edd7e1b1fd48e/locales/zh_CN/LC_MESSAGES/duckduckgo.po#L123-L126

Could you please update this and the other places where "DuckDuckGo" is translated?

**Comment by DenishKakadiya** [comment]
Done all required changes

File: locales/gu_IN/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -70,10 +70,9 @@ msgid "%s%s%s Billion Searches"
 msgstr ""
 
 msgctxt "noresults"
-msgid ""
```
Comment (andrey-p) [inline locales/gu_IN/LC_MESSAGES/duckduckgo.po]: This `msgid` still needs to be present. I think this is why CI is failing.

**Comment by andrey-p** [comment]
One more small comment. Thanks!

**Comment by jbarrett** [comment]
@andrey-p I went ahead and made the `msgid` change - should be good now.

Many thanks @DenishKakadiya!

**Pull request merged**